daemon/daemon_windows.go

```diff
@@ -103,3 +104,33 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	// TODO Windows
 	return nil, nil
 }
+
+func (daemon *Daemon) RegisterLinks(container *Container, hostConfig *runconfig.HostConfig) error {
+	// TODO Windows. Factored out for network modes. There may be more
+	// refactoring required here.
+
+	if hostConfig != nil && hostConfig.Links != nil {
```

Comment:
```go
if hostConfig == nil || hostConfig.Links == nil {
    return nil
}
```

Because the rest of the code is the import stuff, nice to have one less indentation